packages/shared/src/components/ShareMobile.tsx

```diff
@@ -16,23 +16,24 @@ import { useAnalyticsContext } from '../contexts/AnalyticsContext';
 export interface Props {
   post: Post;
   link: string;
-  share: (post: Post) => void;
+  share: () => void;
 }
 
 export function ShareMobile({ post, share, link }: Props): ReactElement {
-  const [copying, copyLink] = useCopyPostLink(link);
+  const [copying] = useCopyPostLink(link);
+
   const { trackEvent } = useAnalyticsContext();
 
   const onShare = () => {
     trackEvent(postAnalyticsEvent(AnalyticsEvent.StartShareToSquad, post));
-    share(post);
```

Comment:
I added this for one edge-case that was sync before.
Not sure where exactly, but maybe we refactored that part already. (just making sure it doesn't break)

Reply:
I think this was using a different share function before that required the post, now it's using the usePostContent share that doesn't need one. Checked for collections, squads and normal posts.

P.s this share/copy thing is also a mess 😭